specs/zrok.yml

```diff
@@ -121,6 +121,30 @@ paths:
         500:
           description: internal server error
 
+  /resetToken:
+    post:
+      tags:
+        - account
+      operationId: resetToken
+      parameters:
+        - name: body
+          in: body
+          schema:
+            properties:
+              emailAddress:
+                type: string
+      responses:
+        200:
+          description: token reset
+          schema:
+            properties:
+              token:
+                type: string
+        404:
+          description: account not found
+        500:
+          description: internal server error
+
```

Comment:
Don't forget the `security` stanza.

Reply:
Definitely test the API endpoint to make sure that the authentication is working (using `curl`).